src/ua.skrypnyk.hw10/Square.java

```diff
@@ -0,0 +1,45 @@
+package src.ua.skrypnyk.hw10;
+
+import java.util.Scanner;
+
+//Perimeter and area should be calculated
+public class Square extends Shape{
+    private double perimeter;
+    private double area;
+    private double side = -4;
+
+    public Square(double perimeter, double area) {
```

Comment:
Area and perimeter should be calculated
Side should be initialized via constructor  
public Square(double side)